pkg/kstatus/polling/statusreaders/common.go

```diff
@@ -209,3 +209,15 @@ func errIdentifierToResourceStatus(err error, identifier object.ObjMetadata) (*e
 		Error:      err,
 	}, nil
 }
+
+func errWouldExceedContextDeadline(err error) bool {
```

Comment:
- rename: `isRateLimiterContextDeadlineExceeded`
- add a method comment explaining why we need this and what it does
- link to the problem in the commit: https://cs.opensource.google/go/x/time/+/refs/tags/v0.10.0:rate/rate.go;l=276

Reply:
@karlkfi https://github.com/kubernetes-sigs/cli-utils/pull/635/commits/31ee857a41bc4f740ea5a87076357455d9e0f79a looks ok?